docs/src/test/java/docs/http/javadsl/server/testkit/TestKitWithActorTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.http.javadsl.server.testkit;
+
+//#testkit-actor-integration
+import akka.actor.testkit.typed.javadsl.TestProbe;
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.javadsl.Adapter;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+
+import akka.http.javadsl.testkit.TestRoute;
+import akka.http.javadsl.testkit.TestRouteResult;
+import org.junit.Test;
+
+public class TestKitWithActorTest extends JUnitRouteTest {
+
+    @Test
+    public void returnPongForGetPing() {
+        ActorSystem<Void> system = Adapter.toTyped(system());
```

Comment:
Maybe add a comment on this line? 

The amount of code included is quite big, so highlighting the line of code (and maybe the import above too) that you are referring to in the paragraph before the snippet will help locate the important bits.

+1 to including the complete code, though!